test/functional/3rd-party-repo-add/3rd-party-repo-add-negative.bats

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+@test "TRA003: Negative test, repo add usage" {
+
+	run sudo sh -c "$SWUPD 3rd-party add $SWUPD_OPTS"
+	assert_status_is "$SWUPD_INVALID_OPTION"
+	expected_output=$(cat <<-EOM
+			Error: The positional args: repo-name and URL are missing
+			*.*
+	EOM
+	)
+	assert_regex_in_output "$expected_output"
+
+	run sudo sh -c "$SWUPD 3rd-party add test-repo1 $SWUPD_OPTS"
+	assert_status_is "$SWUPD_INVALID_OPTION"
+	expected_output=$(cat <<-EOM
+			Error: The positional args: repo-URL is missing
+			*.*
+	EOM
+	)
+	assert_regex_in_output "$expected_output"
+
+	run sudo sh -c "$SWUPD 3rd-party add test-repo1 www.abc.com junk_positional $SWUPD_OPTS"
```

Comment:
this test is using an invalid URL, it is missing the protocol, http, https?

Reply:
Good idea: Thanks will change in next push